### PR TITLE
feat(doc): provide basic example for scss with webpack

### PR DESCRIPTION
**Pull request by seanforyou23**
## Description
This PR makes a few small enhancements to the main readme, namely adding an example of how to configure a webpack build to be able to consume patternfly sass. It ended up being very simple after I figured out how to make it work, but it became obvious that a simple mention in our documentation for how to do this was badly missing.

It's kinda related to https://github.com/patternfly/patternfly-next/issues/720, not directly - but treating the same problem anyway. There just wasn't a good example of how to do this using webpack, a widely adopted module bundler. Instead of making developers painfully figure this out on their own, let's give them a simple example of how to do it.

Also, I added the language type onto various code snippets in the readme so that, as a developer, I get syntax highlighting. Without this, all of the text in the readme is white, which isn't so easy on the eyes.

## Changes

* Include example of webpack configuration
* Include example of how to set required scss vars
* Specify language for various code snippets in the readme

## Link to rawgit and/or image

https://github.com/seanforyou23/patternfly/tree/readme-update-sass-dist#using-webpack

screen shot 2018-10-10 at 5 08 25 pm

screen shot 2018-10-10 at 5 09 22 pm

## PR checklist

- [ ] **Cross browser**: works in IE9
- [ ] **Cross browser**: works in IE10
- [ ] **Cross browser**: works in IE11
- [ ] **Cross browser**: works in Edge
- [ ] **Cross browser**: works in Chrome
- [ ] **Cross browser**: works in Firefox
- [ ] **Cross browser**: works in Safari
- [ ] **Cross browser**: works in Opera
- [ ] **Responsive**: works in extra small, small, medium and large view ports.
- [x] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,43 @@ exclude: [
 **Patternfly now supports Sass natively!**
 Sass is included in the `dist/sass` directory. Just add `node_modules` to your build tool's Sass include paths then `@import 'patternfly/dist/sass/patternfly';` in your Sass to get started!
 
+#### Using Webpack?
+There are two touch points for integratign patternfly sass, one in your webpack conrfig, and another in your sass. Below is an example module rule for loading patternfly .scss files using webpack
```
Comment (mcoker) [inline README.md]: `integratign` and `conrfig` are misspelled. I think it should read something like

> There are two touch points for integrating patternfly sass: one in your webpack config, and another in your sass.

Comment (mcoker) [reply README.md]: oh and add a period after `files using webpack`

**Review by mcoker — DISMISSED** [review]
woohoo! LGTM other than a couple of misspellings.

**Review by jeff-phillips-18 — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -61,6 +61,43 @@ exclude: [
...
 
+#### Using Webpack?
+There are two touch points for integrating patternfly sass: one in your webpack config, and another in your sass. Below is an example module rule for loading patternfly .scss files using webpack.
+
+```javascript
+module: {
+  rules: [
+    {
+      test: /\.scss$/,
+      use: [
```
Comment (TheRealJon) [inline README.md:72]: Can we boil this down to the most simplistic example possible? Are the `style-loader` and `css-loader` parameters necessary?

**Review by TheRealJon — DISMISSED** [review]
Looks good! Just the one comment.

**Review by jeff-phillips-18 — APPROVED** [review]

**Comment by seanforyou23** [comment]
Anything I can do to move this one forward? I realize there's plenty more work we could do around this, but a merge here at least gets us back to where the docs lead users down a happy path.

**Review by TheRealJon — APPROVED** [review]

**Comment by TheRealJon** [comment]
@seanforyou23 I can't merge this, but it looks good to me.

**Pull request merged**